extensions/wikia/UserProfilePageV3/Masthead.class.php

```diff
@@ -488,18 +491,20 @@ private function doRemoveFile() {
 	 *
 	 * @return bool
 	 */
-	public function fileExists() {
+	private function fileExists() {
 		wfProfileIn( __METHOD__ );
 
-		global $wgAvatarsUseSwiftStorage;
+		global $wgAvatarsUseSwiftStorage, $wgAvatarsUseService;
 		if ( !empty( $wgAvatarsUseSwiftStorage ) ) {
 			$swift = $this->getSwiftStorage();
 
 			$avatarRemotePath = $this->getLocalPath();
 			$res = $swift->exists( $avatarRemotePath );
-		} else {
```

Comment:
as there is no `else` therefore no default case, please initialise to silent `Notice: Undefined variable: res in extensions/wikia/UserProfilePageV3/Masthead.class.php on line 512` or assert that at least one of `$wgAvatarsUseSwiftStorage` and `$wgAvatarsUseService` is not empty.